CMake/mingwcc64.toolchain.cmake

```diff
@@ -0,0 +1,21 @@
+SET(MINGW_CROSS TRUE)
+
+SET(CROSS_PREFIX "/usr" CACHE STRING "crosstool-NG prefix")
+
+SET(CMAKE_SYSTEM_NAME Windows)
+
+# workaround
+list(APPEND CMAKE_CXX_IMPLICIT_INCLUDE_DIRECTORIES "${CROSS_PREFIX}/x86_64-w64-mingw32/include")
+
+SET(CMAKE_C_COMPILER   "x86_64-w64-mingw32-gcc")
+SET(CMAKE_CXX_COMPILER "x86_64-w64-mingw32-g++")
+set(CMAKE_RC_COMPILER "x86_64-w64-mingw32-windres")
+set(CMAKE_STRIP "${CROSS_PREFIX}/x86_64-w64-mingw32/bin/strip")
```

Comment:
This toolchain file is debian/ubuntu-specific.
What do you think about borrowing the CMake toolchain files from SDL?
https://github.com/libsdl-org/SDL/blob/main/build-scripts/cmake-toolchain-mingw64-x86_64.cmake